onnxruntime/test/python/onnxruntime_test_python.py

```diff
@@ -869,6 +870,149 @@ def test_session_with_ortvalue_input(ortvalue):
 
             # The constructed OrtValue should still be valid after being used in a session
             self.assertTrue(np.array_equal(ortvalue2.numpy(), numpy_arr_input))
+    def testNumpyToCudaCpu(self):
```

Comment:
The number of tests added doesn't seem to reflect all the python changes. For example, do we need tests to exercise run_with_ort_values() ?

Reply:
Will fix the line. run_with_ort_values() is tested in another file.